### DISCUSSION TITLE
Is it possible to override month and week day names in the calendar view?

### DISCUSSION THREAD

**Discussion by jsyrjala** [wojtekmaj/react-datetime-picker#416] [Q&A] [answered]
I'd like to use Finnish locale to get correct 24-hour clock, date/time field separators etc. But to have all the texts in English, at least month and week day names. 

If I set `locale="fi"` then everything works but the month and week day names are in Finnish. Is it somehow possible to override the month/weekday names?

**Comment by wojtekmaj** [comment, marked as answer]
Yes, you can use formatters like `fornatMonth` which are specified in [React-Calendar README](https://github.com/wojtekmaj/react-calendar). The props will be passed to the calendar.
